packages/api/src/auth/decoders/azureActiveDirectory.ts

```diff
@@ -0,0 +1,49 @@
+import jwt from 'jsonwebtoken'
+import jwksClient from 'jwks-rsa'
+
+export const verifyAzureActiveDirectoryToken = (
```

Comment:
Do you need to export this?

Reply:
Fixed in https://github.com/jeliasson/redwood/commit/0a9d8e922ff37ca10dd620e82cb57606c6f41e56